modules/dns-helper/dns_helper_test.go

```diff
@@ -0,0 +1,383 @@
+package dns_helper
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var publicDomainNameservers = []string{
```

Comment:
Add comment indicating that these are the public nameservers for gruntwork.io, and should be updated if that ever changes.

Reply:
Removed unnecessary specific function DNSDoWithRetryE after implementing new generic DoWithRetryInterfaceE
Added comment about public nameservers for gruntwork.io